main/sdl2/APKBUILD

```diff
@@ -26,10 +28,15 @@ build() {
 		--disable-esd \
 		--disable-nas \
 		--disable-rpath \
-		--with-x
+		--with-x \
+		--enable-video-directfb
 	make
 }
 
+check() {
+    [ "$($builddir/sdl2-config --version)" == "$pkgver" ]
```

Comment:
This is a library, right? Then such check is quite useless, so just disable it with comment (`options="!check"  # upstream doesn't provide tests`).

Reply:
Wait, I thought that would be better than no tests - at least we know that one binary in the package can be executed, so if the compiler was broken or some other fundamental error, it would get caught. But this only counts for programs, that are part of the main program we're compiling, not some program in `$pkgname-dev`?